engine/src/windowing/window.cpp

```diff
@@ -132,6 +133,14 @@ namespace gen
 		log.error("GLFW Error: {} - {}", error, description);
 	}
 
+	void Window::callback_window_size(GLFWwindow * window, int width, int height)
+	{
+		auto * const self = static_cast<Window *>(glfwGetWindowUserPointer(window));
+		self->m_extent.x  = width;
+		self->m_extent.y  = height;
```

Comment:
Just FYI: you can initialize the whole vec in one shot: `self->m_extent = {width, height}`. (Assuming `mim::vec2` supports such initialization.)

Reply:
It does, did not think about doing that!